### PR TITLE
chore: improve installer config documentation

### PR DESCRIPTION
**Pull request by xavierchanth**
**- What I did**

Updated the configuration files for srvd and sshnpd with better documentation and made them easier to configure.

**- How I did it**

**- How to verify it**

**- Description for the changelog**
chore: improve installer config documentation

### CODE REVIEW

File: packages/dart/sshnoports/bundles/shell/systemd/sshnpd.service [hunk 1]
```diff
@@ -2,24 +2,47 @@
 Description=Ssh No Ports Daemon
 After=network-online.target
 
-# Make this unit fail if sshd isn't started first
-Requisite=sshd.service
+# Uncomment the following line to make this unit fail if sshd isn't started first
+; Requisite=sshd.service
```
Comment (xavierchanth) [inline packages/dart/sshnoports/bundles/shell/systemd/sshnpd.service:6]: Both # and ; make valid comments in systemd unit files, so I used `;` to represent items that can be uncommented

**Review by cconstab — DISMISSED** [review]
looks great..

File: packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh [hunk 2]
```diff
@@ -2,14 +2,25 @@
...
+# To see all available options, run `sshnpd` with no arguments.
 
-sleep 10; # allow machine to bring up network
+manager_atsign="@example_client" # MANDATORY: Manager/client address (atSign)
+device_atsign="@example_device"  # MANDATORY: Device address (atSign)
+device_name="default"            # Device name
+user="$(whoami)"                 # MANDATORY: Username
+# s="-s" # Uncomment if you wish the daemon to update authorized_keys to include public keys sent by authorized manager atSigns
+# u="-u" # Uncomment if you wish to have the daemon make various information visible to the manager atsign - e.g. username, version, etc - without the manager atSign needing to know this daemon's device name
+v="-v" # Uncomment to enable verbose logging
```
Comment (gkc) [inline packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh]: This line is not commented

File: packages/dart/sshnoports/bundles/shell/headless/sshnpd.sh [hunk 3]
```diff
@@ -2,14 +2,25 @@
...
+# To see all available options, run `sshnpd` with no arguments.
 
-sleep 10; # allow machine to bring up network
+manager_atsign="@example_client" # MANDATORY: Manager/client address (atSign)
+device_atsign="@example_device"  # MANDATORY: Device address (atSign)
+device_name="default"            # Device name
+user="$(whoami)"                 # MANDATORY: Username
+# s="-s" # Uncomment if you wish the daemon to update authorized_keys to include public keys sent by authorized manager atSigns
+# u="-u" # Uncomment if you wish to have the daemon make various information visible to the manager atsign - e.g. username, version, etc - without the manager atSign needing to know this daemon's device name
+v="-v" # Uncomment to enable verbose logging
```
Comment (gkc) [inline packages/dart/sshnoports/bundles/shell/headless/sshnpd.sh]: Should be commented I assume

File: packages/dart/sshnoports/bundles/shell/headless/root_srvd.sh [hunk 4]
```diff
@@ -2,13 +2,18 @@
...
-# internet_address="127.0.0.1"
+# Configuration of srvd service
+# This unit script is a template for the srvd background service.
+# You can configure the service by editing the variables below.
+# This service file covers the common configuration options for srvd.
+# To see all available options, run `srvd` with no arguments.
 
-sleep 10; # allow machine to bring up network
+atsign="@my_rvd"    # MANDATORY: Srvd atSign
+internet_address="" # MANDATORY: Public FQDN or IP address of the machine running the srvd
```
Comment (gkc) [inline packages/dart/sshnoports/bundles/shell/headless/root_srvd.sh:12]: Should we add the "-v" flag here as well?

File: packages/dart/sshnoports/bundles/shell/headless/srvd.sh [hunk 5]
```diff
@@ -2,13 +2,18 @@
...
-# internet_address="127.0.0.1"
+# Configuration of srvd service
+# This unit script is a template for the srvd background service.
+# You can configure the service by editing the variables below.
+# This service file covers the common configuration options for srvd.
+# To see all available options, run `srvd` with no arguments.
 
-sleep 10; # allow machine to bring up network
+atsign="@my_rvd"    # MANDATORY: Srvd atSign
+internet_address="" # MANDATORY: Public FQDN or IP address of the machine running the srvd
```
Comment (gkc) [inline packages/dart/sshnoports/bundles/shell/headless/srvd.sh:12]: Should we add the "-v" flag here as well?

**Review by gkc — CHANGES_REQUESTED** [review]

File: packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh (hunk 2 above)
Comment (xavierchanth) [reply packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh]: I left verbose uncommented intentionally, I think it should be on by default

Comment (xavierchanth) [reply packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh]: Changed to "Comment to disable"

File: packages/dart/sshnoports/bundles/shell/headless/root_srvd.sh [hunk 6]
```diff
@@ -2,13 +2,19 @@
...
+v="-v"              # Comment to disable verbose logging
+
```
Comment (xavierchanth) [inline packages/dart/sshnoports/bundles/shell/headless/root_srvd.sh:14]: Now says `Comment to disable verbose logging`

File: packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh [hunk 7]
```diff
@@ -2,14 +2,32 @@
...
-sleep 10; # allow machine to bring up network
+manager_atsign="@example_client" # MANDATORY: Manager/client address (atSign)
+device_atsign="@example_device"  # MANDATORY: Device address (atSign)
+device_name="default"            # Device name
+user="$(whoami)"                 # MANDATORY: Username
+v="-v"                           # Comment to disable verbose logging
+
+# Uncomment if you wish the daemon to update authorized_keys to include public
+# keys sent by authorized manager atSigns
+# s="-s"
```
Comment (xavierchanth) [inline packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh:19]: Separated `-s` and `-u` to their own comment blocks

**Review by cconstab — APPROVED** [review]

**Review by gkc — APPROVED** [review]

**Pull request merged**